test/_index.html

```diff
@@ -5,6 +5,6 @@
 <body>
     <h1>Hello world</h1>
 </body>
-<script src="../adoptedStyleSheets.js"></script>
+<script type="module" src="../src/index.js"></script>
```

Comment:
Better to load from dist or set up a script to run this from localhost otherwise this doesn't work from `file://`

Reply:
Oh yeah, I forgot about this part.

Reply:
Fixed

Reply:
I mean, it's still `[type=module]` which won't load from `file://`